### PR TITLE
Convert the Mask RCNN training script to herring

### PR DESCRIPTION
**Pull request by karan6181**
As Title

### CODE REVIEW

File: scripts/instance/mask_rcnn/train_mask_rcnn.py [hunk 1]
```diff
@@ -34,16 +35,7 @@
...
-    import horovod.mxnet as hvd
-except ImportError:
-    hvd = None
-
-try:
-    from mpi4py import MPI
-except ImportError:
-    logging.info('mpi4py is not installed. Use "pip install --no-cache mpi4py" to install')
-    MPI = None
+import herring.mxnet as hvd
```
Comment (rondogency) [inline scripts/instance/mask_rcnn/train_mask_rcnn.py:38]: why are we still using horovod name everywhere?

File: scripts/instance/mask_rcnn/requirements.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+pip install pybind11==2.5.0
+
+# Install NVIDIA COCO
+pip uninstall -y pycocotools
+pip --no-cache-dir --no-cache install \
+    'git+https://github.com/NVIDIA/cocoapi#egg=pycocotools&subdirectory=PythonAPI'
+
+# Install nightly GluonCV for now since changes in Mask RCNN is not compatible with stable pip release
+pip uninstall -y gluoncv
+pip install gluoncv==0.8.0b20200801
```
Comment (rondogency) [inline scripts/instance/mask_rcnn/requirements.sh:10]: is gluon cv gonna fix it in later release, and do we need to build with cython also?

File: scripts/instance/mask_rcnn/train_mask_rcnn.py (hunk 1 above)
Comment (karan6181) [reply scripts/instance/mask_rcnn/train_mask_rcnn.py:38]: The agenda is to minimize the change in existing code as less as possible to make it compatible with herring

File: scripts/instance/mask_rcnn/requirements.sh (hunk 2 above)
Comment (karan6181) [reply scripts/instance/mask_rcnn/requirements.sh:10]: Once we have a 0.8.0 stable release or atleast a tag, we can use that. For now, it doesn't exist

Comment (karan6181) [reply scripts/instance/mask_rcnn/requirements.sh:10]: Added a stable version in a comment

**Review by rondogency — APPROVED** [review]
LGTM!

**Pull request merged**